https://github.com/coreutils/gnulib/commit/0cc39712803ade7b2d4b89c36b143dad72404063

From 0cc39712803ade7b2d4b89c36b143dad72404063 Sun Oct 18 00:00:00 2020
From: Bruno Haible  <bruno@clisp.org>

obstack: Fix a clang warning.
* lib/obstack.c (print_and_abort): Mark as __attribute_noreturn__.

diff --git a/lib/obstack.c b/lib/obstack.c
index 6e82da054c..e6475a2a09 100644
--- a/lib/obstack.c
+++ b/lib/obstack.c
@@ -326,7 +326,7 @@ int obstack_exit_failure = EXIT_FAILURE;
 #   include <libio/iolibio.h>
 #  endif
 
-static _Noreturn void
+static __attribute_noreturn__ void
 print_and_abort (void)
 {
   /* Don't change any of these strings.  Yes, it would be possible to add
